apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/BrevService.kt

```diff
@@ -407,12 +408,23 @@ class BrevService(
             brevDataMapping = { ManueltBrevMedTittelData(it.innholdMedVedlegg.innhold(), it.tittel) },
         )
 
+    // EY-4963
+    private fun sjekkOmErAktivitetsipliktsvurderingsBrev(brevkoder: Brevkoder?): Boolean =
+        listOf(
+            Brevkoder.OMSTILLINGSSTOENAD_AKTIVITETSPLIKT_INFORMASJON_10MND_INNHOLD,
+            Brevkoder.OMSTILLINGSSTOENAD_AKTIVITETSPLIKT_INFORMASJON_4MND_INNHOLD,
+        ).contains(brevkoder)
+
     suspend fun ferdigstill(
         id: BrevID,
         bruker: BrukerTokenInfo,
     ) {
         val brev = sjekkOmBrevKanEndres(id)
 
+        if (sjekkOmErAktivitetsipliktsvurderingsBrev(brev.brevkoder)) {
+            throw BrevKanIkkeEndres(brev, "brevkoden er feil, er ${brev.brevkoder}. Denne kan kun endres i aktivitetsplikts flyten")
+        }
+
```

Comment:
Er det ikke denne routen som behandling bruker?

Reply:
Behandling bruker den for andre type brev og det er greit men vi bruker denne `"$resourceUrl/api/brev/sak/${req.sakId}/ferdigstill-journalfoer-og-distribuer"`